packages/cdk8s-plus/src/size.ts

```diff
@@ -1,10 +1,8 @@
-import { Token } from 'constructs';
```

Comment:
For simplicity, I just dropped the entire token handling in the `Size` class. Do we really need it?

Reply:
Okay